sda-commons-server-opentelemetry/src/main/java/org/sdase/commons/server/opentelemetry/http5/client/ApacheHttpClient5TelemetryBuilder.java

```diff
@@ -0,0 +1,136 @@
+package org.sdase.commons.server.opentelemetry.http5.client;
+
+import io.dropwizard.logback.shaded.errorprone.annotations.CanIgnoreReturnValue;
```

Comment:
Is that a problem when we want to contribute the code to OTEL? I'm sure they don't use Dropwizard. I'd also be fine with not using the annotation.

Reply:
I had to do that because they use a Google annotation, that we don't want to use in our code. When we send them the PR I will rollback to the default annotation